README.md

```diff
@@ -177,6 +177,26 @@ bird:
       pullPolicy: IfNotPresent
 ```
 
+#### bird configwatcher
+
+To observe the bird configs and applying new configs automatically without restarting BIRD, [bird-configwatcher](https://github.com/openvnf/bird-configwatcher) can be used.
+
+To enable `bird-configwatcher` or change images or annotations, change the following parameteres:
+
+```yaml
+bird:
+  birdConfigWatcher:
+    enabled: true # default is false
+    service:
+      annotations:
+        prometheus.io/scrape: "true"
+        prometheus.io/port: "8000"
```

Comment:
These annotations are strictly related to the Prometheus, and it depends on Prometheus configuration how metrics are scrapped - by annotations or some monitors. So I would remove them from the readme.

Reply:
Thanks, removed it.